app/views/layouts/_header.html.erb

```diff
@@ -7,6 +7,7 @@
         <li><%= link_to "Cart", cart_path %></li>
         <% if logged_in? %>
           <li><%= link_to "Sell Item", new_item_path %></li>
+          <li><%= link_to "Receipts", receipts_path(user_id: current_user.id) %></li>
```

Comment:
Don't put this in the header but in the users#show view. This isn't something that we need or want to always be present. The user shouldn't have to spend a lot of time looking for it, but it doesn't need to be part of the header.